src/components/List.js

```diff
@@ -1,4 +1,19 @@
-export const List = ({ data, columns }) => {
+import { Pagination } from "./Pagination";
+
+import logo from "../giphy.webp";
+
+export const List = ({
+  data,
+  columns,
+  total,
+  onPageChange,
+  currentPage,
+  nextPage,
```

Comment:
These 2 functions are not needed. Use onPageChange for handling next and previous page changes

Reply:
Done.